apps/builder/app/builder/features/sidebar-left/sidebar-left.tsx

```diff
@@ -43,6 +43,12 @@ export const SidebarLeft = ({ publish }: SidebarLeftProps) => {
     setActiveTab("none");
   });
 
+  useEffect(() => {
+    if (isClientSettingsLoaded) {
+      $isAiCommandBarVisible.set(clientSettings.isAiCommandBarVisible);
+    }
+  }, [isClientSettingsLoaded, clientSettings.isAiCommandBarVisible]);
```

Comment:
This logic looks a little weird considering client settings are stored in atom too. And this could be just a computed from it.

Reply:
client settings is an atom?